packages/components/src/slot-fill/index.js

```diff
@@ -33,6 +33,7 @@ export function createSlotFill( name ) {
 
 	const SlotComponent = ( props ) => <Slot name={ name } { ...props } />;
 	SlotComponent.displayName = name + 'Slot';
+	SlotComponent.__unstableName = name;
```

Comment:
We need a better way to reference the slot in the `useSlot` call. I added an unstable name for now, but I would be happier if we could use `SlotComponent` as an argument:

```js
const { Slot } = createSlotFill( 'Name' );

useSlot( Slot );
````